components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -741,10 +742,11 @@ def generic_start_worker(
         mounts.clp_home,
         mounts.data_dir,
         mounts.logs_dir,
-        mounts.input_logs_dir,
     ]
     if worker_specific_mount:
         necessary_mounts.extend(worker_specific_mount)
+    if StorageType.FS == clp_config.logs_input.type:
+        necessary_mounts.append(mounts.input_logs_dir)
```

Comment:
Nit:

```suggestion
    if StorageType.FS == clp_config.logs_input.type:
        necessary_mounts.append(mounts.input_logs_dir)
    if worker_specific_mount:
        necessary_mounts.extend(worker_specific_mount)
```

Just so the generic mounts are added first before worker-specifc mounts.